hyperdata/package.json

```diff
@@ -26,7 +26,7 @@
     "lint:fix": "eslint --cache --ext .ts,.js,.mjs,.cjs . --fix && prettier -c src test -w",
     "prepack": "pnpm run build",
     "release": "pnpm test && changelogen --release && npm publish && git push --follow-tags",
-    "test": "vitest run"
```

Comment:
this will leave vitest in watch mode - not preffered as it cannot run in CI

Reply:
vitest able to detect which one development mode or CI
- https://vitest.dev/guide/features.html#watch-mode
- https://github.com/kodadot/workers/blob/21fb3d7f7e74ff00fe50c1fb1022f08c5d24fba9/.github/workflows/test.yml#L22
- https://github.com/kodadot/workers/blob/21fb3d7f7e74ff00fe50c1fb1022f08c5d24fba9/services/image/package.json#L9